oaknational/Oak-Web-Application
# [CUR-466] Connect live curriculum data to test harness (Demo)

## Description

Music year: 2020

- This change uses the existing MVs to create a UI that uses live data for the curriculum download document.
- Currently the data isn't in the exact same format that the visualiser uses (because Sonali is refactoring that area). It is in the same format returned from the MV.
- This is really just the initial scaffolding that can be adjusted as we format the data into the best structure for the document.

In addition, as part of a separate ticket we will need new MVs to support the cycle 2 records, as this only shows published data. I also think that may be an opportunity to improve the existing MVs by making them candidates to replace the existing MVs.

## How to test

1. Go to {owa_deployment_url}/teachers/curriculum/docx-poc/select
2. Type Science or Maths etc
3. You should see dropdown of curriculum options.
4. 'Go to download page' will take you to a slug that loads the data for that selection.
5. There you can add a template file and generate a document based on the live data (although this won't change the document just yet, other tickets will add functionality)

## Checklist

- [x] Added or updated tests where appropriate **(Not applicable for test harness)**
- [x] Manually tested across browsers / devices **(Not applicable for test harness)**
- [x] Considered impact on accessibility **(Not applicable for test harness)**
- [x] Design sign-off **(Not applicable for test harness)**
- [x] Approved by product owner **(Not applicable for test harness)**
- [x] Does this PR update a package with a breaking change **(No)**
